### PR TITLE
comp-3043 - Cleanup remote WPML config files

### PR DESCRIPTION
**Pull request by amElnagdy**

### CODE REVIEW

File: avada/wpml-config.xml [hunk 1]
```diff
@@ -204,35 +204,15 @@
         <shortcode>
             <tag>fusion_builder_column</tag>
             <attributes>
-                <attribute>background_image</attribute>
-                <attribute type="link">link</attribute>
```
Comment (dgwatkins) [inline avada/wpml-config.xml]: Are you sure this line should be removed?
I think it should stay, please confirm @amElnagdy

File: avada/wpml-config.xml [hunk 2]
```diff
@@ -342,11 +314,9 @@
         <shortcode>
             <tag>fusion_lightbox</tag>
             <attributes>
-                <attribute>title</attribute>
-                <attribute type="link">href</attribute>
```
Comment (dgwatkins) [inline avada/wpml-config.xml]: same here

File: avada/wpml-config.xml (hunk 1 above)
Comment (amElnagdy) [reply avada/wpml-config.xml]: @dgwatkins it's duplicated. The shortcode is duplicated.

File: avada/wpml-config.xml (hunk 2 above)
Comment (amElnagdy) [reply avada/wpml-config.xml]: These attributes don't exist. I added the correct ones.

**Pull request merged**